### PR TITLE
Typing env etc fixes

### PR DESCRIPTION
**Pull request by scop**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -21,6 +24,7 @@ deploy:
   on:
     tags: true
     python: 3.7  # prevent deploying multiple times (an therefore causing deploys to fail)
+    condition: -z $TOXENV
```
Comment (aequitas) [inline .travis.yml]: can you add a comment on what problem this lines solves?

Comment (scop) [reply .travis.yml]: Sure, done.

Docs on `condition`: https://docs.travis-ci.com/user/deployment#conditional-releases-with-on

Comment (scop) [reply .travis.yml]: I'm not able to test it, but the intent is for the deployment to kick in when running the "normal" 3.7 build (I assume $TOXENV is empty there), not (also) the typing one.

Comment (aequitas) [reply .travis.yml]: LGTM

**Pull request merged**